### DISCUSSION TITLE
Create a distinction between the page name and name of the page in the table of contents on the left

### DISCUSSION THREAD

**Discussion by TheSevenPens** [GitbookIO/community#428] [Feature Requests]
Today a page's title shows up in the table of contents on the left. This is a good and reasonable default behavior.

The problem is that while I love having more descriptive titles for a page, because the full title is used in the TOC on the left, it creates for some rather large bulky entries.

Example:
![image](https://github.com/GitbookIO/community/assets/98374109/8c18b2d7-7837-40c6-9acf-29200f2dd2a6)

What would be better is to allow me to select a shorter piece of text for that document to be used in the TOC. This would allow the TOC to look nicer, avoid repetition of common title elements,  and be easier to navigate while still providing users with the ability to have long descriptive titles.

This is how for example, the documentation for Microsoft works. See the example below. The page title is "What is Azure Synapse Data Explorer? (Preview)". That page in the TOC appears as "Data Explorer Overview"

![image](https://github.com/GitbookIO/community/assets/98374109/51ef17b5-6bfd-4e8d-9409-6b032345a6e1)

**Comment by lukejfuller** [comment]
This would be very helpful, I've found this limitation frustrating.

**Comment by clairechabas** [comment]
This is now available :)

Yu can now set a "link title" for a page which will be used for the page's table of contents entry and all other relative links (page footer navigation, page cards, mentions...). 

This lets you set a different more descriptive title for the page itself (which will remain the title used for the page automatically computed URL segment as well).

See our updated docs [here](https://gitbook.com/docs/creating-content/content-structure/page#page-link-title).

**Reply by TheSevenPens** [reply]
@clairechabas Thanks so much to the GitBook team for delivering this. It has a big impact for me - making my TOC easy-to-navigate and attractive.
